bezier-rs/lib/src/lib.rs

```diff
@@ -536,6 +536,40 @@ impl Bezier {
 			.filter(|&point| utils::dvec2_approximately_in_range(point, min, max, max_abs_diff).all())
 			.collect::<Vec<DVec2>>()
 	}
+
+	/// Returns a list of lists of points representing the hull points for all iterations at the point corresponding to `t` using de Casteljau's algorithm
+	/// the ith element of the list represents the points of the ith iteration
```

Comment:
Capitalize the start of sentences and add periods at the end of comments. Also, it would likely be beneficial to add some explanation about what a hull point is.

Reply:
Yes, thank you (I still don't know what a hull is in this context so I haven't reviewed the PR yet because I don't know what it does).

Also to elaborate on the capitalization/period rules: for doc comments (with `///`), always use complete sentences. For single-sentence inline comments (`//`), it's okay to omit the period since there aren't multiple sentences and it's more of a note, but still capitalize the first letter. If there are multiple sentences in an inline comment, use sentences with periods just like doc comments.